.circleci/config.yml

```diff
@@ -30,7 +33,8 @@ jobs:
             . venv/bin/activate
             pip install --install-option="--no-cython-compile" cython
             pip install numpy
-            pip install nose mpmath argparse Pillow codecov matplotlib Sphinx==1.7.2
+            pip install nose mpmath argparse Pillow codecov matplotlib "Sphinx<2.1"
+            pip install pybind11
```

Comment:
Surely the `pybind11` addition shouldn't be needed for this branch, but I probably won't remove unless there's a more serious change needed on this branch at this stage, since it isn't really harmful for a CI check only.